distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml

```diff
@@ -36,7 +36,7 @@
         <bundle>mvn:ddf.test.itests/test-itests-common/${project.version}</bundle>
 
         <bundle>mvn:ddf.test.thirdparty/restito/${project.version}</bundle>
-        <feature>broker-app</feature>
+        <feature>camel-sjms</feature>
```

Comment:
The TestMessageBroker class uses the camel-sjms feature as a runtime dependency of the test. We are using it similar to Restito